### ISSUE TITLE
AttributeError: 'tuple' object has no attribute 'lower'

### ISSUE THREAD

**Issue by mm-johnpeterson** [powerschell/AMP4e#1]
`Traceback (most recent call last):
  File "./amp.py", line 42, in <module>
    parameters = pika.URLParameters(amqp_url)
  File "/usr/lib/python2.7/site-packages/pika/connection.py", line 737, in __init__
    if url[0:4].lower() == 'amqp':
AttributeError: 'tuple' object has no attribute 'lower'
`

When I try to run the script, thank you for it by the way, I receive this error regardless of which python version I try. 

Just to make sure this isnt a problem on my end, is 
EVENT_STREAM_NAME = ''
the Queue Name:.... event_stream1234 
that is generated during the event stream creation? OR is it the full amqps:// that is provided?

**Comment by powerschell** [comment]
Hi mm-johnpeterson,

Sorry for the late response, this is the first python script I've ever really done and shared..... Try updating line 39 with your user pw:
 
amqp_url = 'amqps://{user_name}:PUT YOUR USER PW HERE@{host}:{port}'.format(
    **e['amqp_credentials'])

and, yep, event stream name is the queue name! I also just updated it today, so make sure to grab the latest version. I've confirmed it's working with python 3.6. I also have the script setup as a service on my Sumo Logic collector to make sure it gets restarted if it ever times out and it's working pretty nicely for my environment! 

Let me know if that works for you,
Paige
